pkgs/development/r-modules/default.nix

```diff
@@ -1016,6 +1016,14 @@ let
       '';
     });
 
+    quarto = old.quarto.overrideDerivation (attrs: {
+      postPatch = ''
+        substituteInPlace "R/quarto.R" \
+          --replace "path_env <- Sys.getenv(\"QUARTO_PATH\", unset = NA)" "path_env <- '${pkgs.quarto}'" \
```

Comment:
On reviewing this again I think it would be better to simply patch it to `unset = '${lib.getBin pkgs.quarto}/bin/quarto'` which would allow users to still use the environment variable should they wish.